README.md

```diff
@@ -9,8 +9,8 @@ This is a to do list implemented using webpack
 
 ## Screenshot
 
```

Comment:
- [ ] Let's try to be thorough in our documentation as it can be of great benefit to users. You have two titles in your README. Kindly fix that.

  ![richard-title](https://user-images.githubusercontent.com/36542986/154211813-1c89e1cb-7133-40eb-af72-06b229c84725.jpg)

- [ ] Your live demo link is a dummy link. Kindly remove it or add the appropriate link.
- [ ] I'd suggest you are thorough with your app setup. After users clone the repo, kindly include instructions on how to actually run the app. ie `npm install` to install packages, then `npm start` (or `npm run build`) to run the app.

  ![richard-runapp](https://user-images.githubusercontent.com/36542986/154212527-e486baba-808a-433b-af92-3d6a7a49bd75.jpg)

- [ ] **[OPTIONAL]** You probably don't need this section. The initial `npm install` from app set up will take care of them.
- [ ] The luxon package is not part of this project. Kindly remove it.

  ![richard-install](https://user-images.githubusercontent.com/36542986/154212881-c1a8d4d6-7b84-43b3-80b8-e7e80b0cfd43.jpg)

- [ ] Here, **Run tests** is not descriptive of this section. Kindly fix that. I think you meant **Run Linters**

  ![richard-tests](https://user-images.githubusercontent.com/36542986/154213168-470fc999-3703-4e06-8c15-dca966fcbe1c.jpg)